packages/space-opera/src/components/best_practices/styles.css.ts

```diff
@@ -75,7 +75,7 @@ export const arButtonCSS: CSSResult = css`
   left: 50%;
   transform: translateX(-50%);
   white-space: nowrap;
-  bottom: 132px;
+  bottom: 16px;
```

Comment:
What are the visual changes that this change is making? Here and below?

Reply:
These were copied from the WebXR example, which has then really high off the bottom to make room for the carousel, but since that's not here, I decided to put them back down so they're less obtrusive.